docker/testdata/test_grass_session.py

```diff
@@ -1,18 +1,16 @@
-# Import GRASS Python bindings
-# https://github.com/zarch/grass-session
-# pip install grass-session
+# Import GRASS GIS Python bindings (requires 8-4+) and test r.in.pdal
+
+# PYTHONPATH=$(grass --config python-path) python
```

Comment:
I hope we will manage to remove the need to use workarounds like this one day...